src/components/side-panel/panel-header.js

```diff
@@ -297,7 +297,7 @@ function PanelHeaderFactory(SaveExportDropdown, CloudStorageDropdown) {
                       if (item.dropdownComponent) {
                         showExportDropdown(item.id);
```

Comment:
Should we ever have an action item that both shows a dropdown and does something on click? If not, then I can update this to return after this statement.

Reply:
My 2 cents: either-or is a reasonable assumption, weird to do both.